packages/@guidesmiths/cuckoojs-cli/package.json

```diff
@@ -48,13 +48,14 @@
   "scripts": {
     "build": "tsc -p tsconfig.build.json",
     "start": "node dist/src/cli.js",
-    "test": "npm run build && jest src/**/*.spec.ts",
+    "test": "jest src/**/*.spec.ts",
+    "test:dev": "npm run build && jest src/**/*.spec.ts",
     "lint": "eslint --ext .ts .",
     "lint:fix": "eslint --fix --ext .ts ."
   },
   "dependencies": {
     "@angular-devkit/schematics-cli": "^14.2.3",
-    "@guidesmiths/cuckoojs-schematics": "^0.0.2",
+    "@guidesmiths/cuckoojs-schematics": "*",
```

Comment:
I guess that with this lerna automatically knows the dependency tree and first publishes the `cuckoojs-schematics` package and then the `cuckoojs-cli` package, Is that right?

Reply:
Correct

Reply:
To be more precise, I think Lerna handles the dependencies when doing the build.